services/121-service/src/payments/fsp-integration/intersolve-voucher/services/intersolve-voucher.service.ts

```diff
@@ -414,14 +414,25 @@ export class IntersolveVoucherService {
       return;
     }
 
-    // Do not update transaction (or create event) on success callback on initial message
+    // Do not update transaction (or create event) on success callback on initial message //##TODO discuss this again (don't update transaction, but do create event?)
```

Comment:
Let's discuss this indeed

Reply:
Let’s but we can leave it outside of this pr